force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +56,21 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+
+        // if (age >= 18) {
+        //     System.debug('True');
+        //     Return true; 
+
+        // }else {
+        //     System.debug('False');
+        //     Return false; 
+
+        // }
+
+        Boolean result = age >= 18 ? true : false;
```

Comment:
Excellent use of a ternary operator here. However, logical comparisons will always return a boolean so `Boolean result = age >= 18;` would have the same result

Reply:
Thnak you! i will make sure to remember that.